XWG8AW/Domain/User.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace XWG8AW.Domain
+{
+    internal class User
+    {
+        public User() { }
```

Comment:
Miért van 3 konstruktora?

Reply:
Azokat még az elején hoztam létre mielőtt a játékmenettel foglalkoztam volna és akkor még nem tudtam, hogy melyikkel is akarnék dolgozni. Utólag a default konstruktor elég lenne, mert a property-ket setter-rel állítom be.